scripts/src/admin/translations.ts

```diff
@@ -123,18 +131,19 @@ async function getCsvRowData() {
 	let rows: CsvRow[] = [];
 	allOffers.forEach(offer => {
 		const offerId = offer.id;
+		const offerName = offer.name;
 		const offerContent: OfferContent = allContent.filter(obj => obj.offerId === offerId)[0];
```

Comment:
You can change this to check for name and remove the `offerId` var completely.

Reply:
much more clear to use offerId as pkey